tutorial03_traceable_layers/layerlib/CMakeLists.txt

```diff
@@ -15,18 +15,18 @@ include_directories(${SRC_DIR}/include
                     ${SRC_DIR}/layers
                     ${SRC_DIR}/loader)
 add_library(layer_utils STATIC
-        ${LAYER_DIR}/common/vk_layer_config.cpp
-        ${LAYER_DIR}/common/vk_layer_extension_utils.cpp
-        ${LAYER_DIR}/common/vk_layer_utils.cpp)
+        ${SRC_DIR}/layers/vk_layer_config.cpp
```

Comment:
building is meant to be traceable into layers; provide cmake as an option might be handy for applications that have adopted cmake + android studio support, also be ready for other components of layers/utilities when they move to cmake build.